src/redux/ReduxInjector.ts

```diff
@@ -1,18 +1,24 @@
 import { Store } from 'redux';
 import { Injector } from '@dojo/widget-core/Injector';
 
+export interface ReduxInjectorPayload<S, O> {
+	extraOptions?: O;
```

Comment:
nit: should the mandatory properties come first? just feels odd having the `extraOptions` list first here.

Reply:
I tend to order everything alphabetically, but will change the ordering to required => optional.